src/renderer/modules/components/Loader.tsx

```diff
@@ -21,7 +21,6 @@ export type LoaderType = React.ComponentType<LoaderProps> & {
   Type: typeof TYPES;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
-export default (await waitForModule(filters.bySource('"wanderingCubes"')).then((mod) =>
+export default await waitForModule(filters.bySource('"wanderingCubes"')).then((mod) =>
   Object.values(mod).find((x) => typeof x === "function"),
-)) as LoaderType;
+);
```

Comment:
Type here needs to be put back. Think this is getting caused by some ESLint version thing but idk.

Reply:
honestly I don't see any problem without that, as the type is specified when exporting in index.ts

Reply:
@FedeIlLeone If we import it directly inside the mod, it needs to have types